dev-ruby/net-scp/net-scp-1.2.1-r1.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-USE_RUBY="ruby23 ruby24 ruby25 ruby26"
+USE_RUBY="ruby25 ruby26 ruby27"
```

Comment:
Why is this needed? net-scp 3.0.0 already has ruby27 support, and from what I can see that is the version that vagrant needs?